### PR TITLE
Add timer to refresh trust page stats

### PR DESCRIPTION
**Pull request by phillmac**
An untested basic idea.
Run the load_trust_statistics() every minute.

### CODE REVIEW

**Comment by synctext** [comment]
OK to test

File: TriblerGUI/widgets/trustpage.py [hunk 1]
```diff
@@ -85,11 +85,16 @@ def __init__(self):
...
         self.blocks = None
         self.byte_scale = 1024 * 1024
+        self.timer = None
 
     def initialize_trust_page(self):
         vlayout = self.window().plot_widget.layout()
         self.trust_plot = TrustPlotMplCanvas(self.window().plot_widget, dpi=100)
         vlayout.addWidget(self.trust_plot)
+        
+        self.timer = QtCore.QTimer()
```
Comment (devos50) [inline TriblerGUI/widgets/trustpage.py]: It seems that the GUI tests are failing. Please use:

```
from PyQt5.QtCore import QTimer
```

and simply use the `QTimer` object 👍

File: TriblerGUI/widgets/trustpage.py [hunk 2]
```diff
@@ -85,11 +85,16 @@ def __init__(self):
         self.statistics = None
         self.blocks = None
         self.byte_scale = 1024 * 1024
+        self.timer = None
```
Comment (devos50) [inline TriblerGUI/widgets/trustpage.py]: Could you please rename this variable to something more descriptive? Like `refresh_timer`?

**Comment by phillmac** [comment]
I'm unsure why Pylint is failing, I tried moving the import above the others but it made no difference.
Any guidance here much appreciated.

**Pull request merged**